cips/cip-18.md

```diff
@@ -38,49 +38,62 @@ The following API is proposed for the standardised gas estimation service.
 
 ```proto
 service GasEstimator {
-    rpc EstimateGasUsage(EstimateGasUsageRequest) returns (EstimateGasUsageResponse) {}
-    rpc EstimateGasPrice(EstimateGasPriceRequest) returns (EstimateGasPriceResponse) {}
+    rpc EstimateFee(EstimateFeeRequest) returns (EstimateFeeResponse) {}
+    rpc EstimateFeeAndGas(EstimateFeeAndGasRequest) returns (EstimateFeeAndGasResponse) {}
```

Comment:
As per discussing with @vgonkivs, what do you think of adding a linear formula similar to PFBs that can be used by light clients as a backup if there is an issue with the gas estimation endpoint? It could be aggressive but at least ensures transaction inclusion most of the time.

Reply:
We could add it at least for the most popular txs.

Reply:
I wanted to include this in the ADR for celestia-node: PFBs would use the linear model, all other transactions would request the gas estimate

Reply:
In the future we could add a model for other popular transactions like MsgSend, MsgTransfer, MsgClaim etc.